### PR TITLE
refactor: adjust max screen with to fit all phone sizes

### PR DESCRIPTION
**Pull request by d4v3d4vE**
- changed the max screen width to from 428 to 504 to fit all phone sizes
   -  https://yesviz.com/viewport/  biggest phone viewport width in this list: 504
- adjusted the buttons width to match with the bottom design on all devices
- fixed the TabBar look on iPhone 14 Pro Max and any other devices with a bigger viewport width than 428

| Before (iPhone 14 Pro Max) | After (iPhone 14 Pro Max) |
| ------------- | ------------- |
| Simulator Screenshot - iPhone 14 Pro Max - 2023-09-04 at 11 47 27 | Simulator Screenshot - iPhone 14 Pro Max - 2023-09-04 at 11 46 59 |
| Simulator Screenshot - iPhone 14 Pro Max - 2023-09-04 at 11 47 39 | Simulator Screenshot - iPhone 14 Pro Max - 2023-09-04 at 11 47 14 |

SVA-1151

### CODE REVIEW

File: src/config/consts.js [hunk 1]
```diff
@@ -99,7 +99,7 @@ export const consts = {
 
   DIMENSIONS: {
     // the max screen size we want to render full screen
-    FULL_SCREEN_MAX_WIDTH: 428
+    FULL_SCREEN_MAX_WIDTH: 430
```
Comment (ardasnturk) [inline src/config/consts.js:102]: Are there devices with bigger screens? This update is only available for the Pro Max. But have you checked for larger screen devices on the Android platform?

Comment (d4v3d4vE) [reply src/config/consts.js:102]: you're right, there are plenty devices which require a bigger viewport than the iPhone 14 Pro Max
[Here](https://yesviz.com/viewport/) is a list of viewports for other devices

Comment (donni106) [reply src/config/consts.js]: did you check some landscape screens? can you share some screenshots of them as well please

Comment (d4v3d4vE) [reply src/config/consts.js]: yes I also checked them, there are no changes cause 504 as viewport width is still smaller than pretty much all up to date smartphones are in landscape mode 
|iPhone 14 Pro Max) |iPhone SE|
| ------------- | ------------- |
| Simulator Screenshot - iPhone 14 Pro - 2023-09-11 at 11 41 25 | Simulator Screenshot - iPhone SE (3rd generation) - 2023-09-11 at 11 49 26 |

Comment (donni106) [reply src/config/consts.js]: the tab navigation would be interesting in comparison, like in the portait screenshots. can you share them as well please.

**Review by donni106 — APPROVED** [review]

**Review by ardasnturk — APPROVED** [review]
🎉

**Pull request merged**